src/WordPress/Hooks.php

```diff
@@ -274,6 +274,20 @@ protected function isPluginSpecificCacheEnabled()
             && $cacheSettingValue !== 'off';
     }
 
+    protected function isAutomaticPlatformOptimizationEnabled()
+    {
+        $cacheSettingObject = $this->dataStore->getPluginSetting(\CF\API\Plugin::SETTING_AUTOMATIC_PLATFORM_OPTIMIZATION);
```

Comment:
where is SETTING_AUTOMATIC_PLATFORM_OPTIMIZATION defined?

Reply:
seems to be missing in https://github.com/cloudflare/Cloudflare-WordPress/blob/9ea34a61cd2a2b9b96561f159144f0b3ed1286f9/vendor/cloudflare/cloudflare-plugin-backend/src/API/Plugin.php#L17

Reply:
I have added it in https://github.com/cloudflare/cloudflare-plugin-backend/pull/48